src/pages/orders/screens/Orders/Orders.tsx

```diff
@@ -47,7 +48,12 @@ const Orders = () => {
             <AwarenessBanner />
             <OrdersTableHeader fromDate={fromDate} setFromDate={setFromDate} setToDate={setToDate} toDate={toDate} />
             {!isDesktop && <Divider />}
-            <OrdersTable data={data} isActive={isActive} isLoading={isLoading} loadMoreOrders={loadMoreOrders} />
+            <OrdersTable
+                data={data}
+                isActive={isActive}
+                isLoading={isLoading || (isFetching && isActive)}
```

Comment:
I only allowed the loader to show when isFetching for isActive because we have infinite loading. Usually the users wont have as many active orders at a time, but for past orders there can be a lot, so everytime it fetched it would show the loader and bring the user back to the top.

Also the main reason i also passed isFetching here for loading is because the timer was shown immediately without checking if there is some new data being fetched. It will recognise that a new item has been added but because it doesn't wait until the data loads, it will show the previous timer value until we change or reload the page. So this will prevent it as we will properly show the loader on the page.